### ISSUE TITLE
Welcoming newcomers to metauni via Roblox

### ISSUE THREAD

**Issue by blinkybool** [metauni/general#8]
It seems that metauni becomes a little more visible on Roblox when we have events, and people wander in who don't know what we're doing. This can lead to disruptions during scheduled seminars, and while some people just need to be kicked, I suspect others just might not be aware that we are holding an event.

- Perhaps we can show an on-join message while an event is running which briefly explains what's going on and a polite request for them to either listen quietly or explore the world. 
- It should also make the new-comer feel welcome, and that they're not intruding.
- Can we include a metauni.org link or is this against the rules? Or can we prompt them to check out the metauni group on Roblox?
- The metauni Roblox group should probably have a description explaining what we are. Can this have a metauni.org link? I see we have a twitter link. If we can't link metauni.org, could the description say something like "find out more on our twitter page"?
- If there's no easy way to find metauni.org from Roblox, should we start putting the schedule in the metauni group description?

**Comment by dmurfet** [comment]
Yes I think this is an excellent idea. Ben suggested that there be a welcome message that appears the first time you join a metauni world. It might be a little complicated to have this show once globally (i.e. only once across all metauni worlds) but I think it suffices for now to have it show once per world (we detect this by putting a DataStore tag). 

I don't think this message needs to detect whether or not an event is running, which is its own problem - it suffices to show it to people who have never visited metauni before.

We cannot include a metauni.org link in any message. I do not know about advertising Roblox groups.

Yes to more information in the Roblox group. We cannot include arbitrary URLs but we can use the "social links" to add links. The Twitter link was already there, I had some problem just now adding YouTube but I did succeed in adding a Discord link.

Re: the schedule in the metauni group description. I'm against that for the piddling reason that it sounds like even more work for me on a weekly basis ;)

**Comment by dmurfet** [comment]
Bumping this, as per the discussion in the Discord.

I think we also need a more refined "kick" system. I want something that a speaker can manage during a talk, where they can quickly put up a "warning" message to a user (something like, thanks for dropping by, but please don't disrupt the talk) without having to stop a talk and type in the chat. Maybe this should also teleport the user back to the starting spot, in case they are in the way.
